org.mixedrealitytoolkit.input/Tests/Runtime/BasicInputTests.cs

```diff
@@ -279,6 +279,8 @@ public IEnumerator StatefulInteractableSmoketest()
         /// Simple smoketest to ensure basic gaze-pinch selection functionality.
         /// </summary>
         [UnityTest]
+        [Ignore("Temporarily ignoring this test because MRTKGazeController is not fully migrated to XRI 3 yet.")]
```

Comment:
Can you open a bug to track reenabling these.  So we don't forget

Reply:
Sure, done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/issues/744 , resolving.